README.md

```diff
@@ -60,6 +60,22 @@ when Ecto gets more complex field selection support we will go further to only q
 
 You will need to handle filtering yourself, the filter is just a map with key=value.
 
+## Configuration
+
+There are several options available through configuration.
```

Comment:
Sounds good to me. Will make this change + add a note on the reasoning when I have a bit of time later today.